lisdf/components/model.py

```diff
@@ -186,14 +190,21 @@ class Link(StringConfigurable):
     collisions: List[Geom] = field(default_factory=list)
     visuals: List[Geom] = field(default_factory=list)
     sensors: List[Sensor] = field(default_factory=list)
-    model: Optional["Model"] = None
-
-    mjcf_configs: Optional[Dict[str, str]] = None
-    sdf_configs: Optional[Dict[str, str]] = None
-    # sites: List[Site] = None
 
-    def set_model(self, model):
-        self.model = model
+    def to_sdf(self) -> str:
+        collision_str = "\n".join([c.to_sdf() for c in self.collisions])
+        visual_str = "\n".join([v.to_sdf() for v in self.visuals])
+        return f"""<link name="{self.name}">
+    {self.pose.to_sdf()}
+    {self.inertial.to_sdf() if self.inertial else ""}
+    <collision>
+        {collision_str}
```

Comment:
Not that it matters much, but does `collision_str` and `visual_str` maintain the indenting you want? Not sure how it plays out with multiline f-strings (more interested, nothing needs to be changed)

Reply:
Nope it wont. I was trying to quickly craft something... Let me reformat them haha

Reply:
Fixed! The indent_text helper function does the magic lol